docs/release_notes/release_notes.md

```diff
@@ -12,6 +12,30 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
 {% endif %}
 
+## Notable Features
+
+### Puppet 4 Support
+
+Puppet 4 is now supported by Katello. For more information, visit [the Puppet upgrade page](http://www.katello.org/docs/nightly/upgrade/puppet.html)
+
+### Content Host Subscriptions Bulk Actions
+
+Users can now attach Subscriptions to multiple Content Hosts using the UI. 
+
+### File type repository updates
+
+In Katello 3.1, the file repository type was added for storing generic files in Katello. The 3.2 release brings with it the ability to add those file repositories to a content view. Now you can you snapshot your generic files into views and push them through environments just as you would for RPM or Puppet content. This can be used to create a production set of Python packages, Rubygems, tarballs or even Debian repositories. In addition, an API has been added for viewing file content in your Katello just as you would any other content type.
+
+### Searchable ostree repos
+
```

Comment:
Adds a new Content -> OSTree Branches to enable one to search on available OSTree Branches across different content view/lifecycle environments.